keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -75,8 +75,8 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
   - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
 - [x] (R) Graduation criteria is in place
   - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
-- [ ] (R) Production readiness review completed
-- [ ] (R) Production readiness review approved
+- [x] (R) Production readiness review completed
```

Comment:
Nit: above `(R) Ensure GA e2e tests meet requirements for [Conformance Tests` should also be checked

Reply:
I confirmed that, thanks.